qemu/tests/win_heavyload.py

```diff
@@ -122,26 +122,27 @@ def add_option(cmd, key, val):
         logging.info("heavyload cmd: %s" % start_cmd)
         session.sendline(start_cmd)
         if not loop_session_cmd(session, check_running_cmd):
-            raise error.TestError("heavyload process is not started")
+            test.error("heavyload process is not started")
 
         sleep_before_migration = int(params.get("sleep_before_migration",
                                                 "0"))
         time.sleep(sleep_before_migration)
 
-        error.context("Verify vm is alive", logging.info)
+        error_context.context("Verify vm is alive", logging.info)
         utils_misc.wait_for(vm.verify_alive,
-                            timeout=test_timeout, step=steping)
+                            timeout=test_timeout*60, step=steping)
```

Comment:
A better way is to remove L118, and correct L114 to:
```
logging.warn("Heavyload timeout is too small (%ss), force set to 60s", test_timeout)
```

Reply:
@luckyh  for `start_cmd` in L119,  it use test_timeout in minutes, and for `utils_misc.wait_for` in L132, it use test_timeout in seconds,  So i think we can not remove L118.

Reply:
Then we can set it to the right unit in L119, so that we can get consistent unit expression in script.

Reply:
yeah,  updated, could you help check it again?  thanks !